core/src/replay_stage.rs

```diff
@@ -480,7 +480,7 @@ impl ReplayStage {
             Self::reset_poh_recorder(
                 &my_pubkey,
                 &blockstore,
-                &working_bank,
+                working_bank,
                 &poh_recorder,
                 &leader_schedule_cache,
             );
```

Comment:
smaller bp fix would be to just `drop(working_bank)` here?

Reply:
^ this would also fix the bug with a smaller diff. I am fine with either approach for BP

Reply:
Correct, an explicit `drop()` call after `Self::reset_poh_recorder()` would do the trick too (at the cost of 1-to-1 mapping of commits between master and v1.14)

Reply:
at this point, increasing bp conflicts in 1.14 to reduce likelihood of more backports to 1.14 is a good thing in my book. :sweat_smile: 

i'm fine either way